src/vellum/workflows/nodes/displayable/bases/inline_prompt_node/node.py

```diff
@@ -45,7 +46,7 @@ class BaseInlinePromptNode(BasePromptNode, Generic[StateType]):
     blocks: ClassVar[List[PromptBlock]]
 
     # The functions/tools that a Prompt has access to
-    functions: Optional[List[FunctionDefinition]] = OMIT
+    functions: Optional[List[Union[FunctionDefinition, Callable]]] = None
```

Comment:
I know we stole the `OMIT` idea from Fern, but this value is not falsy so it fails the `if self.functions` check on line 68 below.

Since we're already using optional here, I'd rather default to `None` as we did here, or to our undefined repr `UNDEF`, which _is_ falsy.

Reply:
I'm generally a fan of replacing our `OMIT`s with `UNDEF`